test/test_utils/test_reporting.py

```diff
@@ -43,7 +43,8 @@ class TestReportGenerator:
     Class to generate test report files
     """
 
-    ALLOWED_SINGLE_GPU_TESTS = ("telemetry", "test_framework_and_cuda_version_gpu", "test_curand_gpu")
+    ALLOWED_SINGLE_GPU_TESTS = ("telemetry", "test_framework_and_cuda_version_gpu", "test_curand_gpu",
```

Comment:
are we also doing for cpu as well?

Reply:
This is for tests that we allow to be run on single gpu instances - cpu does not need whitelisting.